services/graphql-server/src/graphql/definitions/platform/content/index.js

```diff
@@ -472,6 +478,28 @@ input ContentEventDatesMutationInput {
   allDay: Boolean = false
 }
 
+"Content Id to update with UserRegistration payload"
+input ContentUserRegistrationMutationInput {
+  "Content Id"
+  id: Int!
+  "Payload to update"
+  payload: ContentUserRegistrationMutationPayloadInput!
+}
+
+"User Registration fields to update on content by id"
+input ContentUserRegistrationMutationPayloadInput {
+  "Is Registration Required"
```

Comment:
Clean up the comments to be more declarative.. e.g. "Whether registration is required to view the content"

And for dates.... "The date when the registration requirement begins" etc